### PR TITLE
To add py39 jobs and remove py27 jobs from Cisco ASA

### PR DESCRIPTION
**Pull request by justjais**
Depends-On: https://github.com/ansible/ansible-zuul-jobs/pull/1522

To add py39 jobs and remove py27 jobs from Cisco ASA

### CODE REVIEW

**Comment by justjais** [comment]
@Gomathiselvi-S @goneri Can you plz approve and merge the PR, I've put non-voting for py integration jobs which is currently failing coz of authentication failure coz of public key authentication, it might be coz of [this](https://lists.opendev.org/pipermail/service-discuss/2021-October/000291.html).
As this is blocking the release for Cisco ASA for AAP 2.2, I've put the jobs as non-voting as it's unrelated to integration/test failures

File: playbooks/ansible-network-asa-appliance/pre.yaml [hunk 1]
```diff
@@ -1,6 +1,11 @@
 ---
 - hosts: controller
+  gather_facts: true
   tasks:
+    - name: Print ansible_host
```
Comment (GomathiselviS) [inline playbooks/ansible-network-asa-appliance/pre.yaml:5]: same as above

File: playbooks/ansible-network-asa-appliance/files/bootstrap.yaml [hunk 2]
```diff
@@ -9,6 +9,10 @@
     ansible_user: admin
 
   tasks:
+    - name: Print appliance ansible_host
```
[the remark this answers is not in the document]
Comment (justjais) [reply playbooks/ansible-network-asa-appliance/files/bootstrap.yaml:12]: ah yes

**Review by GomathiselviS — APPROVED** [review]

**Comment by justjais** [comment]
recheck

**Comment by goneri** [comment]
recheck

**Pull request merged**